src/MapComponent.js

```diff
@@ -72,4 +72,17 @@ export default class MapComponent extends Component<any, any, any> {
     const el = this.leafletElement
     if (el) el.fire(type, data)
   }
+
+  getInstanceOptions (props = {}) {
+    const pane = props.pane || this.context.pane
```

Comment:
`pane` should always be injected from the context, not props, that could lead to weird behaviors I think.